src/v1/components/data-display/Chip/Chip.tsx

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import MUIChip, { ChipProps as MUIChipProps } from "@mui/material/Chip";
+
+export type ChipProps = Omit<
+  MUIChipProps,
+  | "avatar"
+  | "children"
+  | "classes"
+  | "clickable"
+  | "color"
+  | "component"
+  | "deleteIcon"
+  | "icon"
+  | "skipFocusWhenDisabled"
+  | "sx"
```

Comment:
Maybe using the styled approach might be better at it would provide access to the theme as well. Which maybe be useful for consistency and maybe will encourage others to not use custom properties and stick to the theme wherever possible

Reply:
https://telicent.atlassian.net/browse/TELPD-22